drake/systems/plants/RigidBody.cpp

```diff
@@ -2,6 +2,9 @@
 #include "RigidBody.h"
 #include <stdexcept>
 
+#include "drake/util/drakeGeometryUtil.h"
+#include "drake/util/testUtil.h"
```

Comment:
I will remove the inclusion of testUtil.h by removing the `operator==()` methods from this PR. I'm hoping that decreasing the size of this PR will make it easier to merge into the master branch.